dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -9,6 +9,10 @@
 
 #include <winsock.h>
 
+#include <atlsimpcoll.h>
+
+CSimpleArray<LANSTATUSUI_CONTEXT *> g_pContextArray;
```

Comment:
Why introduce a global array here?
There seems to be a window (with state) for each context thing, so why store info that is available already in a global array, to retrieve the info that is available already from that global array?

Reply:
You're right. Fixed